docs/source/examples/custom_saving.rst

```diff
@@ -0,0 +1,86 @@
+Saving custom fields to the output file
+=======================================
+
+Given the flexibility of the pyDeltaRCM model to modifications via hooks and subclassing, it is necessary that the variables saved to the output netCDF file are similarly customizable.
+Fortunately, the use of subclasses and hooks itself enables flexible setting of gridded variables to be saved as figures, as well as customization of the fields saved to the netCDF file as both variables, and as metadata.
+
+To customize the figures and variables that are saved, the hook ``hook_init_output_file`` should be used to subclass the pyDeltaRCM model.
+
+When adding a model attribute to the list of grids to save as figures, be sure to append the actual *attribute name* to the list of figures to save.
+For example, ``self._save_fig_list.append('active_layer')`` will properly indicate that figures of the active layer should be saved, however ``self._save_fig_list.append('activelayer')`` will not work because the name of the model attribute is ``active_layer`` not ``activelayer``.
+
+When adding variables or metadata to be initialized and subsequently saved in the output netCDF, the key-value pair relationship is as follows.
+The key added to ``self._save_var_list`` is the name of the variable as it will be recorded in the netCDF file, this *does not* have to correspond to the name of an attribute in the model.
+To add a variable to the metadata, a key must be added to ``self._save_var_list['meta']``.
+The expected value for a given key is a list containing strings indicating the model attribute to be saved, its units, the variable type, and lastly the variable dimensions (e.g., ``['active_layer', 'fraction', 'f4', ('total_time', 'length', 'width')]`` for the active layer).
```

Comment:
reiterate here which dimensions are available? It's just that the three right ('total_time', 'length', 'width')? 

Reiterate that the dimension names have to match *exactly* to one of these three.

Reply:
Good point, added an "important" note